code/modules/atmospherics/components/binary_devices/massive_gas_pump.dm

```diff
@@ -86,7 +86,7 @@
 	if (power_draw >= 0)
 		last_power_draw = power_draw
 
-		power_machine.draw_power(power_draw)
+		power_machine.draw_power(power_draw * 0.001)
```

Comment:
Seemingly redundant multiplication, you're just multiplying a value by 1000 only to divide it back later. I'm assuming pump_gas doesn't care what unit it's being fed.